src/BodyFunctionality/BodyPartInterface.java

```diff
@@ -0,0 +1,13 @@
+package BodyFunctionality;
+
+import javafx.scene.paint.Color;
```

Comment:
So with this interface, we're trying to skeleton out _everything_ that a bodypart could possible do - so every function and public object in bodypart should be in here.